destiny-app/src/components/hooks/fetchMain.js

```diff
@@ -0,0 +1,12 @@
+// import axios from 'axios';
```

Comment:
Why is all of this commented out for now?

Reply:
I am testing code to make things work. You will see some code that has been commented out, incase I need to use it later. Also some code is there for debugging and testing...

Reply:
Sounds good. What will axios do once un-commented? (Just curious, don't let it hold up the merge)

Reply:
It's the most popular HTTP client for Node.js, it's like requests for Python.

Reply:
Got it, thanks.